reference/price_lists.v3.yml

```
@@ -850,7 +850,12 @@ paths:
       tags:
         - Price Lists Records
       summary: Get All Price List Records
-      description: Returns a list of *Price List Records* associated with a *Price List*.
+      description: |-
+        Returns a list of *Price List Records* associated with a *Price List*.
+
+        **Notes**
+        * Up to 10 simultaneous GET requests are supported with this API. Running more than the allowed concurrent requests in parallel on the same store will result in a `429` status error and your additional requests will fail.
```

```suggestion
        * Supports up to 10 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
```
`concurrent requests in parallel` is redundant.